swap_meet/item.py

```diff
@@ -0,0 +1,29 @@
+class Item:
+
+    def __init__(self, category = None, condition = None):
+        if category == None:
+            self.category = ""
+        else:
+            self.category = category
+
+        if condition == None:
+            self.conditionn = 0
+        else:
+            self.condition = condition
```

Comment:
one thing we can do here is let the default parameters do the same work the if-else statements are doing. We can combine them:

```suggestion
    def __init__(self, category = "", condition = 0):
        self.category = category
        self.condition = condition
```

now the parameter will either take in a user's argument or, if left blank, will assign its default value